test/common/assertSnapshot.js

```diff
@@ -24,8 +33,20 @@ function replaceWindowsPaths(str) {
   return common.isWindows ? str.replaceAll(path.win32.sep, path.posix.sep) : str;
 }
 
-function replaceFullPaths(str) {
-  return str.replaceAll('\\\'', "'").replaceAll(path.resolve(__dirname, '../..'), '');
+function replaceWindowsDriveLetter(str) {
+  if (!common.isWindows) {
+    return str;
+  }
+  const currentDriveLetter = path.parse(process.cwd()).root.substring(0, 1).toLowerCase();
+  const regex = new RegExp(`${currentDriveLetter}:`, 'gi');
+  return str.replaceAll('\\\'', "'").replaceAll(regex, '');
+}
+
+function transformCwd(replacement = '') {
+  const cwd = process.cwd();
+  return (str) => {
+    return str.replaceAll('\\\'', "'").replaceAll(cwd, replacement);
+  };
```

Comment:
That looks odd, in what cases do we have to rewrite the CWD?

Reply:
This makes cwd in the error stack as `*`, so that the test snapshots do not capture the cwd.

Reply:
Yeah but the CWD should not appear in stack traces, no?